examples/base/sub-headings.html

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Sub-headings
+category: _base
+---
+<h1>Title</h1>
+<p class="p-heading--three">Sub-heading</p>
```

Comment:
So if an `<h2>` has a subheading, should that be `<p class="p-heading--four">`? Maybe be explicit about using whichever size is two levels below the heading.

Reply:
@matthewpaulthomas you make a good point. Will update 😄 thanks